pyproject.toml

```diff
@@ -122,14 +122,15 @@ reportUnnecessaryTypeIgnoreComment = "error"
 reportUnknownMemberType = "none"
 reportUnknownArgumentType = "none"
 reportUnknownVariableType = "none"
-reportUnknownParameterType = "none"
-reportMissingTypeStubs = "none"
 # These rules don't provide much value
 reportPrivateUsage = "none"
 reportUnnecessaryIsInstance = "none"
 reportUnknownLambdaType = "none"
-# These rules should be progressively enabled 
+# These rules require improved upstream type annotations
+reportMissingTypeStubs = "none"
```

Comment:
Running into typing issues surrounding NetworkX. I want to leave these alone for now, and resolve them in a future PR. Very confusing behavior: my local pyright differs from pylance (due to https://github.com/microsoft/python-type-stubs), which differs from the CI pyright.